blackjax/mcmc/mala.py

```diff
@@ -80,14 +80,14 @@ def transition_energy(state, new_state, step_size):
         """Transition energy to go from `state` to `new_state`"""
         theta = jax.tree_util.tree_map(
             lambda new_x, x, g: new_x - x - step_size * g,
```

Comment:
Should probably update the variable names here.

Reply:
Indeed, should probably exchange `new_x` and `x` here I guess